onvm/onvm_mgr/onvm_stats.h

```diff
@@ -79,6 +79,12 @@
 
 typedef enum { ONVM_STATS_NONE = 0, ONVM_STATS_STDOUT, ONVM_STATS_STDERR, ONVM_STATS_WEB } ONVM_STATS_OUTPUT;
 
+struct onvm_event {
+        const char *type;
```

Comment:
type should be a numeric value, uint8_t would work. 
Then you can define a few different types like ONVM_EVENT_INFO, ONVM_EVENT_CORE_INFO, smth like that